include/bout/region.hxx

```diff
@@ -605,8 +605,8 @@ public:
     return *this;
   }
 
-  /// Return a new region equivalent to *this but with indices contained
-  /// in mask Region removed
+  /// Updates the region so that indices contained in mask Region are
```

Comment:
The previous comment was more correct: this returns a copy of `this` with `maskRegion` removed

Reply:
Apologies for the incorrect change.